src/components/search/Sort/sort.scss

```diff
@@ -0,0 +1,41 @@
+@import "../../../styles/utilities";
+
+.sort {
+  position: relative;
+  font-size: var(--text-size-2);
+  line-height: 1.25;
+
+  label {
+    margin-right: var(--space-1);
+    color: var(--color-text-subtle);
+  }
+
+  [data-store-select] {
+    min-height: var(--min-size);
```

Comment:
We don't it to be too high on the desktop version, I suggest to add this:
`@include media(">=notebook") { height: var(--space-6); }`

Reply:
nice!